File: Guestbook.Web/src/app/api/contracts/review.ts
export class Review {
    id: string;
    created: number;
    comment: string;
    restaurantId: string;
    userName: string;
    email: string;
    reviewRating: number;
    like: number;


    constructor(data?: any) {
        if (data) {
            for (var i in data) {
                this[i] = data[i];
            }
        }
    }

    serialize() {
        return {
            restaurantId: this.restaurantId,
            comment: this.comment,
            email: this.email,
            userName: this.userName,
            reviewRating: this.reviewRating,
            like: this.like,
        }
    }
}

File: Guestbook.WebApi/Wcf.Domain/Review.cs
using GuestBook.Data;
using System;
using System.ComponentModel.DataAnnotations;

namespace GuestBook.Domain
{
    public class Review : IModel
    {
        [Required]
        [StringLength(32, MinimumLength = 1)]
        public string Id { get; set; }

        [Required]
        [StringLength(255, MinimumLength = 1)]
        public string Comment { get; set; }

        [Required]
        public DateTime? Created { get; set; }

        [Required]
        public Restaurant Restaurant { get; set; }

        [Required]
        public string UserName { get; set; }

        [Required]
        public int ReviewRating { get; set; }

        [Required]
        public int Like { get; set; }

        public Review()
        {
            Id = IdentityGenerator.NewId();
            Created = DateTime.UtcNow;
        }
    }
}
